src/js/utils/api.js

```diff
@@ -26,6 +26,10 @@ function status(response) {
 }
 
 function json(response) {
+    if (response.status === 202 || response.status === 204) {
```

Comment:
what does `response.json()` return if not `undefined` or `null`? Or does it throw?